src/attestation/Attestation.ts

```diff
@@ -1,7 +1,7 @@
 /**
- * Attestation provides the KILT ecosystem with handling [[Claim]] objects, which are then written on the [[Blockchain]].
+ * Attestation provides the KILT ecosystem with certifying [[Claim]] objects, which are then written on the [[Blockchain]].
```

Comment:
Sounds a bit bumpy. Maybe "Attestations are used to certify [[Claim]] objects, ..." or similar